scripts/presubmit-tests.sh

```diff
@@ -181,7 +181,7 @@ function default_build_test_runner() {
     | sort | uniq | tr '\n' ' ')"
   for pkg in ${tagged_pkgs}; do
     # `go test -c` lets us compile the tests but do not run them.
-    if ! capture_output "${report}" go test -c -tags="${tags}" ${pkg} ; then
+    if ! capture_output "${report}" go test -c -tags="${tags}" "${pkg}" ; then
```

Comment:
I wasn't sure $pkg is always a single package, so I was worried about this one

Reply:
`tagged_pkgs` is a list of packages so `$pkg` is always a single one.